content/en/about/case-studies/bluecore/index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Bluecore Leverages Istio to Migrate to Kubernetes"
+linkTitle: "Bluecore Leverages Istio to Migrate to Kubernetes"
+quote: "Istio enables an approach that allows you to start building right out of the box." 
+author:
+    name: "Shray Kumar"
+    image: "/img/authors/shray-kumar.jpg"
+companyName: "Bluecore"
+companyURL: "https://bluecore.com/"
+logo: "/logos/bluecore.png"
+skip_toc: true
+skip_byline: true
+skip_pagenav: true
+doc_type: article
+sidebar_force: sidebar_case_study
+type: case-studies
+weight: 30
```

Comment:
@craigbox, do we have anything that talks about the weighting, or do we expect new case studies to be at the end? In this case the weight overlaps with `Bol.com`.

Reply:
@pymander let's make it 51 please

Reply:
Done.